.github/workflows/ci.yml

```diff
@@ -3,6 +3,20 @@ name: CI
 on: [pull_request]
 
 jobs:
+  presubmit:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Python environment
+      uses: actions/setup-python@v1.1.1
+      with:
+        python-version: 3.7
+    - name: Build benchmarks
+      run: |
+        make install-dependencies
```

Comment:
make install-dependencies and source .venv/bin/activate are not needed with make presubmit 
(as it does it automatically). can remove lines 16 and 17.